anaUltraScurve.py

```diff
@@ -210,16 +226,61 @@ def overlay_fit(VFAT, CHAN):
     trimrange_list[event.vfatN][event.vfatCH] = event.trimRange
     pass
 
+# Determine hot channels
+from anautilities import isOutlierMADOneSided
+import numpy as np
 if options.SaveFile:
+    print 'Determining hot channels'
+    masksDead = []
+    masksHot = []
     masks = []
+    for vfat in range(0, 24):
+        trimValue = np.zeros(128)
+        dead = np.zeros(128)
+        for ch in range(0, 128):
+            # Get fit results
+            threshold[0] = scanFits[0][vfat][ch]
+            noise[0] = scanFits[1][vfat][ch]
+            pedestal[0] = scanFits[2][vfat][ch]
+            # Identify dead channels
+            dead[ch] = (threshold[0] == 8 and pedestal[0] == 8)
```

Comment:
I'd like to see how many times the fit fails and if this corresponds with dead channels or just bad fitting.  Please investigate.  We should be sure of the reasoning behind why we need to mask things especially moving toward LS2.